eventmesh/Etcd3EventMeshServer.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const config = require('../broker/lib/config');
```

Comment:
We should have a different config for eventmesh. Let's not use the broker config.

Reply:
Added new config.js and settings.yml. Updated README